apps/BE/src/index.ts

```diff
@@ -0,0 +1,25 @@
+import express from "express";
+import { Request, Response } from 'express';
+import { postgraphile } from "postgraphile";
```

Comment:
why no connect with supabase client?

Reply:
one seems to have limited functionality and the other doesn't + when using prisma it seems like supabase's graphiql doesn't pick up the tables + if we do end up using supabase here then there's no point in setting up a BE which was a "requirement" as requests will go directly via supabase's URL.
Also [pg_graphql](https://github.com/supabase/pg_graphql) has like 3k starts and [postgraphile](https://github.com/graphile/crystal) has like 12k though we could also use hasura or smth else I guess, didn't feel like this is critical as long as its working tbh

I will note though that when instantiating the connection some random warning seems to be thrown. Initially we had some caching issues but once we enabled realtime on the tables the issues no longer appear so I guess you can try and work with that and if we encounter issues we will migrate (GQL will remain the same anyways regardless of the platform)